apps/transport/lib/transport_web/router.ex

```diff
@@ -38,11 +38,13 @@ defmodule TransportWeb.Router do
     plug(:transport_data_gouv_member)
   end
 
-  get("/swaggerui", OpenApiSpex.Plug.SwaggerUI, path: "/api/openapi")
+  scope "/" do
+    pipe_through(:browser)
+    get("/swaggerui", OpenApiSpex.Plug.SwaggerUI, path: "/api/openapi")
+  end
```

Comment:
```suggestion
  scope "/", OpenApiSpex.Plug do
    pipe_through(:browser)
    get("/swaggerui", SwaggerUI, path: "/api/openapi")
  end
```

J'apprends en lisant la [doc](https://hexdocs.pm/phoenix/Phoenix.Router.html#module-scopes-and-resources), que ça accélère les temps de compilation de mettre le nom du module au niveau du scope. 

![image](https://user-images.githubusercontent.com/15341118/209654107-df4c860f-f5ac-4081-8d8f-0d70420be4c3.png)